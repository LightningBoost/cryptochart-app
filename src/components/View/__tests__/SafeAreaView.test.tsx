import React from 'react';
import {render} from '@testing-library/react-native';
import {View} from 'react-native';
import SafeAreaView from '../safeAreaView';

it('should match the snapshot', () => {
  const component = render(
    <SafeAreaView>
      <View />
    </SafeAreaView>,
  ).toJSON();

  expect(component).toMatchSnapshot();
});
