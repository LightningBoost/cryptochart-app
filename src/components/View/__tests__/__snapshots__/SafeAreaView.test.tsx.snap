// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<RNCSafeAreaView
  style={
    Array [
      Object {
        "flex": 1,
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View />
</RNCSafeAreaView>
`;
